applications/nrf5340_audio/Kconfig.sysbuild

```diff
@@ -4,7 +4,37 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
-source "${ZEPHYR_BASE}/share/sysbuild/Kconfig"
-
 config NRF_DEFAULT_IPC_RADIO
 	default y
+
+config AUDIO_DFU_EXTERNAL_FLASH
+	bool "Audio Application DFU using external flash"
+	default n
+	help
+	  Use external flash to store the images downloaded during DFU. This requires the external
+	  flash shield with the MX25R6435F on the SPI bus.
+
+config AUDIO_DFU_INTERNAL_FLASH
+	bool "Audio Application DFU using internal flash"
+	default n
+	help
+	  Use the internal flash to store the images downloaded during DFU. This mode only support
+	  single image mode, i.e. image for application and network must be upgraded separately.
+
+choice BOOTLOADER
+	default BOOTLOADER_MCUBOOT if AUDIO_DFU_EXTERNAL_FLASH || AUDIO_DFU_INTERNAL_FLASH
+endchoice
+
+config SECURE_BOOT
+	default y if AUDIO_DFU_EXTERNAL_FLASH || AUDIO_DFU_INTERNAL_FLASH
+
+config SECURE_BOOT_NETCORE
+	default y if AUDIO_DFU_EXTERNAL_FLASH || AUDIO_DFU_INTERNAL_FLASH
+
+config NETCORE_APP_UPDATE
```

Comment:
PLease add a note here as well about APP<->NET possibly being broken if this is not set.